jbpm-installer/src/test/java/org/jbpm/persistence/scripts/quartzmockentities/QrtzJobDetails.java

```diff
@@ -16,23 +16,53 @@
 
 package org.jbpm.persistence.scripts.quartzmockentities;
 
+import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.Lob;
 
-@Entity(name = "qrtz_job_details")
+import org.hibernate.annotations.Type;
+
+@Entity(name = "QRTZ_JOB_DETAILS")
+@IdClass(QrtzJobDetailsId.class)
 public class QrtzJobDetails {
 
     @Id
-    private Long id;
+    @Column(name = "SCHED_NAME")
+    private String schedulerName;
+
+    @Id
+    @Column(name = "JOB_NAME")
+    private String jobName;
+
+    @Id
+    @Column(name = "JOB_GROUP")
+    private String jobGroup;
+
+    @Column(name = "DESCRIPTION")
+    private String description;
+
+    @Column(name = "JOB_CLASS_NAME")
+    private String jobClassName;
+
+    @Column(name = "IS_DURABLE")
+    @Type(type = "boolean")
+    private Boolean isDurable;
+
+    @Column(name = "IS_NONCONCURRENT")
+    @Type(type = "boolean")
```

Comment:
Do we need these type declarations after we got rid of the types??

Reply:
No, removed all Type annotations